lib/types.ts

```diff
@@ -206,6 +206,10 @@ export interface Config {
     host: string;
     /** The SMTP port to send emails from */
     port: number;
+    /** How to use TLS when connecting: “auto” means “immediate” on port 465, “starttls” otherwise; using `null` is not discouraged as it means “immediate” if port is 465, but opportunistic StartTLS or plain transmission otherwise */
+    tlsMode: null | "auto" | "immediate" | "starttls" | "none";
+    /** Whether to verify the TLS certificate, a string verifies it using that hostname */
```

Comment:
I think this can be clearer.

```suggestion
    /** Whether to verify the TLS certificate. A hostname can be used instead of true/false as well. */
```